### DISCUSSION TITLE
Where can I view the content of comments?

### DISCUSSION THREAD

**Discussion by Memorablea** [community/community#60910] [New to GitHub] [answered]
Where can I view my comments on the Code page?

**Comment by samar19947** [comment]
هنا

المرسل من Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Memorable ***@***.***>
‏‏تم الإرسال: Saturday, July 15, 2023 9:10:19 AM
إلى: community/community ***@***.***>
نسخة: Subscribed ***@***.***>
‏‏الموضوع: [GitHub Community] Where can I view the content of comments? (Discussion #60910)


Where can I view my comments on the Code page?

**Comment by Rohan-San** [comment, marked as answer]
Hey there!
You can view your comments on your Code page as such:

![comment_code](https://github.com/community/community/assets/107957865/4f4f8ddf-9703-4a86-bfb2-1f1dc86c5dfe)

The underlined is the latest description added to the repo.
The encircled symbol takes you to all the comments.
Hope this helped!

**Reply by Memorablea** [reply]
OK

**Reply by queenofcorgis** [reply]
Thank you for answering @Rohan-San and including the helpful screenshot ✅ . Appreciate you marking the answer @Memorablea, I'm sure you're not the only one with this question and by marking you made it easy to share that this solution worked for you.

**Reply by Rohan-San** [reply]
> Thank you for answering @Rohan-San and including the helpful screenshot ✅ . Appreciate you marking the answer @Memorablea, I'm sure you're not the only one with this question and by marking you made it easy to share that this solution worked for you. 

Thank you! Glad to make an effective contribution to GitHub!😊
